.circleci/continue_config.yml

```diff
@@ -0,0 +1,278 @@
+version: 2.1
+orbs:
+  go: circleci/go@1.8.0
+  node: circleci/node@5.2.0
+  slack: circleci/slack@4.10.1
+commands:
```

Comment:
Is there a way to break up this large file? Perhaps using orbs?

Reply:
This looks more complex using circleci than it is with github actions, so I will leave this for future work.